server/api.yaml

```diff
@@ -1561,24 +1678,65 @@ components:
         id:
           type: string
           example: Pv3R416r
-        name: 
+        name:
           type: string
-        email: 
+        email:
           type: string
           example: user@example.com
-        imageUrl: 
+        imageUrl:
           type: string
-          example: "https://images.example.com/image.png"
+          example: 'https://images.example.com/image.png'
+        isActive:
+          type: boolean
+          example: true
+
+    CreateUser:
+      type: object
+      required:
+        - email
+        - name
+      properties:
+        email:
+          type: string
+          format: email
+          example: user@example.com
+        name:
+          type: string
+          example: John Doe
+        imageUrl:
+          type: string
+          example: 'https://images.example.com/image.png'
+        isActive:
+          type: boolean
+          default: true
+          example: true
+
+    UpdateUser:
+      type: object
+      properties:
+        email:
+          type: string
+          format: email
+          example: user@example.com
+        name:
+          type: string
+          example: John Doe
+        imageUrl:
+          type: string
+          example: 'https://images.example.com/image.png'
+        isActive:
+          type: boolean
+          example: true
```

Comment:
Maybe merge it with `CreateUser` because of my previous comments about `UpdateUser` + partial?